plugins/auth/token.js

```diff
@@ -16,7 +16,7 @@ module.exports = () => ({
         notes: 'Generate a JWT for use throughout Screwdriver',
         tags: ['api', 'auth', 'token'],
         auth: {
-            strategies: ['token', 'session', 'auth_token'],
+            strategies: ['session', 'auth_token'],
```

Comment:
I'm still pretty sure we shouldn't be able to refresh a JWT by passing the same JWT, unless this goes through the workflow used to generate the original JWT, e.g. check the auth provider [github] to ensure access hasn't been revoked.